src/main/kotlin/graphics/scenery/DFTParser.kt

```diff
@@ -0,0 +1,132 @@
+package graphics.scenery
+import graphics.scenery.repl.REPL
+import graphics.scenery.utils.RingBuffer
+import org.joml.Vector3f
+import org.lwjgl.system.MemoryUtil
+import java.io.File
+import java.nio.ByteBuffer
+
+/**
+Parses a density functional theory (common simulation method in solid state physics and theoretical chemistry)
+calculation. Can be used to visualize single DFT calculations or DFT-MD (=DFT molecular dynamics simulations).
+
+ * @property[normalizeDensityTo] Defines to which value the density is scaled. This is useful when visualizing more then one DFT calculation at the
+    same time, in order to keep the density visualization consistent. Negative values mean the density is scaled
+    per snapshot. Default is -1.0f.
+ */
+class DFTParser (val normalizeDensityTo: Float = -1.0f){
+    /**  Number of Atoms.*/
+    var numberOfAtoms:Int = 0
+    /** Distance between two grid points in either direction in Bohr.*/
+    var gridSpacings = FloatArray(3) { 0.0f }
```

Comment:
`floatArrayOf(0.0f, 0.0f, 0.0f)` is faster, as it won't incur the lambda overhead for each element you are initialising 👍

Reply:
Same goes for all the lines below 👍

Reply:
I fixed that for the arrays I'm initializing, although for `electronicDensity` I cannot really do that since I am allocating it at runtime, or am I missing something here?